Kth largest element/kth_largest.java.txt

```diff
@@ -0,0 +1,50 @@
+Kth largest element
+
+Given an array A of random integers and an integer k, find and return the kth largest element in the array.
+
+Input Format :
+Line 1 : An integer N i.e. size of the array
+Line 2 : N integers which are elements of the array, separated by spaces
+Line 3 : An integer k
+
+Output Format :
+kth largest element
+Input Constraints :
+1 <= N, Ai, k <= 10^5
+Sample Input 1 :
+6
+9 4 8 7 11 3
+2
+Sample Output 1 :
+9
+Sample Input 2 :
+8
+2 6 10 11 13 4 1 20
+4
+Sample Output 2 :
+10
+
+
+Below is the implementation of the above approach
+import java.util.*;
```

Comment:
Probably okay for now but a future learning for you. Never use `*` while importing packages. Only import the required classes. In this particular use case, Java would import all the classes under `java.util.` and load it in memory. Most of the classes won't be needed for execution of this program thereby consuming unnecessary memory. We should strive towards reducing the memory used by a program under execution.